airbyte-ci/connectors/metadata_service/orchestrator/orchestrator/assets/connector_metrics.py

```diff
@@ -0,0 +1,60 @@
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
+
+import json
+from collections import defaultdict
+from typing import Optional
+
+import sentry_sdk
+from dagster import OpExecutionContext, asset
+from google.cloud import storage
+from orchestrator.logging import sentry
+
+GROUP_NAME = "connector_metrics"
+
+
+@sentry_sdk.trace
+def _safe_read_gcs_file(gcs_blob: storage.Blob) -> Optional[str]:
+    """Read the connector metrics jsonl blob.
+
+    Args:
+        gcs_blob (storage.Blob): The blob.
+
+    Returns:
+        dict: The metrics.
+    """
+    if not gcs_blob.exists():
+        return None
+
+    return gcs_blob.download_as_string().decode("utf-8")
```

Comment:
❓ Any reason to not deserialize jsonl str to list of dict here? Would help catch invalid files earlier.

Reply:
No major reason! Let me change